drivers/SmartThings/harman-luxury/src/disco.lua

```diff
@@ -41,20 +32,12 @@ function Discovery.set_device_field(driver, device)
   local device_cache_value = driver.datastore.discovery_cache[device.device_network_id]
 
   -- persistent fields
-  device:set_field(const.STATUS, true, {
-    persist = true,
-  })
   device:set_field(const.IP, device_cache_value.ip, {
```

Comment:
Does this ever get updated if the IP address of the speaker changes due to DHCP?

Reply:
Good point. I forgot taking this into account when changing to WebSockets. I'll add some logic to handle this scenario as I switch to the lua library's lustre library